src/components/Tabs/Tab.jsx

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styles from "./styles.css";
+
+const Tab = ({
+  onSelect,
+  tabIndex,
+  isActive,
+  className,
+  style,
+  label,
+  count,
+  ...props
+}) => (
+  <li
+    className={`${isActive ? styles.tabActive : styles.tab} ${className}`}
+    style={style}
+    onClick={(e) => {
+      e.preventDefault();
+      onSelect(tabIndex);
+    }}
+    {...props}>
+    <span className={styles.tabLabel} style={style}>
+      {label}
+    </span>
+    {typeof count === "number" && count >= 0 ? (
```

Comment:
This `typeof` verification is unnecessary because the operator `>=` will not return true for non-numeric values.